docs(README): add info for using catppuccin/nix (#174)
catppuccin/gtk: README.md

### For Nix users

> We suggest you uses [catppuccin/nix](https://github.com/catppuccin/nix). 

Reviewer: ```suggestion
We suggest you use [catppuccin/nix](https://github.com/catppuccin/nix). 
```

> Alternatively, the you can use [catppuccin-gtk](https://github.com/NixOS/nixpkgs/blob/master/pkgs/data/themes/catppuccin-gtk/default.nix) from nixpkgs.

Reviewer: ```suggestion
Alternatively, you can use [catppuccin-gtk](https://github.com/NixOS/nixpkgs/blob/master/pkgs/data/themes/catppuccin-gtk/default.nix) from nixpkgs.
```

```nix
{inputs, ...}: {
  imports = [inputs.catppuccin.homeManagerModules.catppuccin];

  gtk = {
    enable = true;
    catppuccin = {
      enable = true;
      flavor = "mocha";
      accent = "pink";
      size = "standard";
      tweaks = [ "normal" ];
    };
  };
}
```

> [!TIP]

> > See [the full documentation](https://github.com/catppuccin/nix/blob/main/docs/home-manager-options.md#gtkcatppuccinenable) for more help.

Reviewer: ```suggestion
> For further information on the options available, see the [the full documentation](https://github.com/catppuccin/nix/blob/main/docs/home-manager-options.md#gtkcatppuccinenable).
```

> > For further information on the options available, see the [the full documentation](https://github.com/catppuccin/nix/blob/main/docs/home-manager-options.md#gtkcatppuccinenable).

Reviewer: ```suggestion
> For further information on the options available, see the [full documentation](https://github.com/catppuccin/nix/blob/main/docs/home-manager-options.md#gtkcatppuccinenable).
```

### For Nix users

> #### Using our flake

Reviewer: ```suggestion
#### Using [catppuccin/nix](https://github.com/catppuccin/nix)
```
